Cargo.toml

```diff
@@ -54,9 +54,10 @@ wit-parser = "0.13.1"
 testresult = "0.3.0"
 
 [features]
-default = ["cli-interactive"]
+default = ["cli-interactive", "keyring"]
 postgres = ["warg-server/postgres"]
 cli-interactive = ["warg-client/cli-interactive"]
+keyring = ["warg-client/keyring"]
```

Comment:
I don't know that we actually need to make this optional for the binary do we? Actually, I didn't really think about this until now: it probably doesn't make sense to have _any_ of these features; what does the `postgres` feature even mean here?

Reply:
The `postgres` feature is for the `warg-server` switches between using an in-memory vs Postgres backing when setting the feature flag. That probably still makes sense for now.

The other `keyring` and `cli-interactive` features might make sense to have the option of compiling without those implementations in the `warg-client`.

Reply:
These features are just for the crate defined in this file: `warg-cli`

Reply:
Ah, you mean we can drop `postgres` in the top-level Cargo.toml but keep it in the `crates/server/Cargo.toml`?

Reply:
Yeah, most of the root `Cargo.toml` is (somewhat confusingly) completely independent of other workspace members.

Reply:
Removing `postgres` from the top-level `Cargo.toml` appears to affect how I run tests. It might affect the test runner, but would need to double check. But doesn't seem urgent.

Reply:
Ok; are the `keyring` and `cli-interactive` features needed?